pimd/pim_nb_config.c

```diff
@@ -114,6 +115,58 @@ static void pim_if_membership_refresh(struct interface *ifp)
 	pim_ifchannel_delete_on_noinfo(ifp);
 }
 
+/*
+ * pim_if_igmp_version_change is called when the igmp version changes
+ * If the igmp version is changed from 3 to 2, delete the 224.0.0.2
+ * membership from the socket so that it won't accept any packets.
+ * If the version is changed 2 to 3, add the membership to the socket.
+ */
+static void pim_if_igmp_version_change(struct interface *ifp)
```

Comment:
Why doesn't this function belong in pim_igmp.c?

Reply:
Fixed